pyproject.toml

```diff
@@ -15,6 +15,7 @@ pydantic = "^1.10.2"
 typer = {extras = ["all"], version = "^0.7.0"}
 beartype = "^0.12.0"
 jinja2 = "^3.1.2"
+funcx = "^1.0.11"
```

Comment:
I think this is a merge artifact that got left in by mistake

Reply:
removed